bedrock/security/templates/security/client-bug-bounty.html

```diff
@@ -82,16 +82,28 @@ <h4 id="security-vulnerability-amounts">Rewards Amount</h4>
     </tr>
   </thead>
   <tbody>
+    <!-- ========================================================= -->
     <tr>
       <td>Highest Impact
-        <ul class="mzp-u-list-styled">
+      </td>
+      <td>$20,000</td>
+      <td>$18,000</td>
+    </tr>
+    <tr>
+      <td style="border-top: 0px">
```

Comment:
Please remove inline CSS styles from this PR. CSS should be in `.scss` files, external to a page's HTML.